packages/action-password-update/src/components/PasswordUpdateDialog/PasswordUpdateDialog.scss

```diff
@@ -1,54 +1,206 @@
-.PasswordUpdateDialog {
+/*****
+* import sass dependencies
+*/
 
-  .oldPassword {
-    margin-bottom: 1.5em;
-  }
+// TODO reduce redundancy
+// TODO get rid of cumbersome imports
+// TODO make it possible to use _variables.scss from customer
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss";
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_opacity.scss";
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_vendor-prefixes.scss";
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_background-variant.scss";
+@import "../../../../../node_modules/font-awesome/scss/_variables.scss";
 
-  .ValidationRules {
-    margin-bottom: 1.5em;
-  }
+$fa-font-path: "../../../../../node_modules/font-awesome/fonts/";
+@import "../../../../../node_modules/font-awesome/scss/font-awesome.scss";
+
+
+/*****
+* tocco-client boilerplate
+* for styling convenience while developing, only used inside monorepo tocco-client
+*/
+
+// TODO replace #root with #tocco-client-development
```

Comment:
Id's has to be unique in html document by convention. On package level we can not assure that. I would like to replace the id by a css class. Semantically is "root" the wrong term, because it indicates the very top level element of a whole, which is only true in a single package. Right now we use this selector for boilerplate styles, which helps while developing but is unwanted in production. There are better class names than my proposal. Do you have preference?